[fbcode] s,\bnp\.bool\b,bool,

igorsugak opened the pull request:
Test Plan: sandcastle and visual inspection

Differential Revision: D49196618




cc @gujinghui @PenghuiCheng @XiaobingSuper @jianyuh @jgong5 @mingfeima @sanchitintel @ashokei @jingxu10 @min-jean-cho @yanbing-j @Guobing-Chen @Xia-Weiwen

Skylion007 commented:
There is a ruff rule for this: https://docs.astral.sh/ruff/rules/numpy-deprecated-type-alias/ Let's enable it to enforce this change codebase wide (it will also handle the replacements for you).

Skylion007 commented:
I think we might also need to bump the min numpy version in our requirements. That's why I haven't enabled this ruff rule yet.

Skylion007 commented:
Actually, we should be fine as those things have been aliases for the most part for a long time.

igorsugak committed:
[fbcode] s,\bnp\.bool\b,bool, (#110253)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/110253

Test Plan: sandcastle and visual inspection

Differential Revision: D49196618
